lti_consumer/lti13.py

```diff
@@ -0,0 +1,187 @@
+from django.conf import settings
+
+from urllib.parse import urlencode
+
+from Crypto.PublicKey import RSA
+from jwkest.jwk import RSAKey
+from jwkest.jws import JWS
+import json
+from jwkest import jwk
+
+
+LTI_BASE_MESSAGE = {
+    # Claim type: fixed key with value `LtiResourceLinkRequest`
+    # http://www.imsglobal.org/spec/lti/v1p3/#message-type-claim
+    "https://purl.imsglobal.org/spec/lti/claim/message_type": "LtiResourceLinkRequest",
+
+    # LTI Claim version
+    # http://www.imsglobal.org/spec/lti/v1p3/#lti-version-claim
+    "https://purl.imsglobal.org/spec/lti/claim/version": "1.3.0",
+
+    # Optional claims - useless ones
```

Comment:
Should I include these optional claims? These are just course and platform metadata.
Ref: http://www.imsglobal.org/spec/lti/v1p3/#optional-message-claims-0

Reply:
I'm wondering if these would be useful at all. Given they are optional, the LTI providers might not even expect them to be provided.

Other considerations is how staff will set this up, if the platform will provide them automatically or not.

Reply:
I've dropped them from the implementation since they are not being used now.